Clarify that the tool runs offline (#983)
semgrep/semgrep: README.md

  </a>
</p>

> Semgrep is a [command-line](#installation) tool for offline static analysis. Use pre-built or custom rules to enforce code and security standards in your codebase. You can try it now with our [interactive live editor](https://semgrep.live/).

Reviewer: I agree with specifying that this is an offline thing.
```suggestion
Semgrep is a [command-line](#installation) tool for offline static analysis. Use pre-built or custom rules to enforce code and security standards in your codebase. You can try it now with our [interactive live editor](https://semgrep.live/editor).
```

Author: thanks @pabloest , good call

Semgrep combines the convenient and iterative style of `grep` with the powerful features of an Abstract Syntax Tree (AST) matcher and limited dataflow. Easily find function calls, class or method definitions, and more without having to understand ASTs or wrestle with regexes.

Visit [Installation](#installation) and [Usage](#usage) to get started.

## Installation

> > Want to skip installation? You can run Semgrep via a live editor at [semgrep.live](https://semgrep.live/).

Reviewer: ```suggestion
> Want to skip installation? You can run Semgrep via our interactive live editor at [semgrep.live](https://semgrep.live/editor).
```
I like the use the word interactive above so matching it here, too.

2. The future of security involves automatically guiding developers towards a “paved road” made of default-safe frameworks (i.e. [React](https://reactjs.org/) or Object-relational Mappers)
3. `grep` isn’t expressive enough and traditional static analysis tools (SAST) are too complicated/slow for paved road automation

> The AppSec, Developer, and DevOps communities deserves a static analysis tool that is fast, easy to use, code-aware, multi-lingual, and open source!

Reviewer 2: typo: `deserve`
```suggestion
The AppSec, Developer, and DevOps communities deserve a static analysis tool that is fast, easy to use, code-aware, multi-lingual, and open source!
```
